scripts/windows/uninstall-wsl.ps1

```diff
@@ -0,0 +1,14 @@
+# //////////////////////////////////////////////////////////////////////
+# uninstall-wsl.ps1
```

Comment:
Is this ever run from anywhere?

Reply:
I've run it to test how rancher-desktop behaves when wsl is uninstalled. But no, nothing in our code base consumes this file.